### PR TITLE
docs(security): add SECURITY.md

### PR DESCRIPTION
**Pull request by Splaktar**
**Does this PR fix a regression since 1.7.0, a security flaw, or a problem caused by a new browser version?** 
No, but it documents the project's Security policy in a standard GitHub-supported way.

**What is the current behavior? (You can also link to an open issue here)**
There is no Security policy in the repo, only on the docs website.

**What is the new behavior (if this is a feature change)?**
There is a Security policy in the repo, in a well known GitHub-supported location.

**Does this PR introduce a breaking change?**
No

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] Fix/Feature: [Docs](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#documentation) have been added/updated
- [ ] Fix/Feature: Tests have been added; existing tests pass

**Other information**:

### CODE REVIEW

**Comment by Splaktar** [comment]
The e2e tests are failing in TravisCI, but obviously not related in any way to a `.md` file being added :)

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+| Version     | Supported          | Status                | Comments                             |
+| ----------- | ------------------ | --------------------- | ------------------------------------ |
+| 1.7.x       | :white_check_mark: | Long Term Support     | Last version to provide IE 8 support |
```
Comment (petebacondarwin) [inline SECURITY.md:7]:
```suggestion
| 1.7.x       | :white_check_mark: | Long Term Support     | See [Long Term Support policy][0] |
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| 1.3.x-1.6.x | :x:                |                       |                                      |
+| 1.2.x       | :warning:          | Security patches only | See [Long Term Support policy][0]    |
```
Comment (petebacondarwin) [inline SECURITY.md:9]:
```suggestion
| 1.2.x       | :warning:          | Security patches only | Last version to provide IE 8 support |
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

**Comment by Splaktar** [comment]
I don't know why I didn't get notified about your comments here. Taking a look at them now.

**Comment by Splaktar** [comment]
Fixed. Not sure how I got that mixed up, thank you!

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+| Version     | Supported          | Status                | Comments                             |
+| ----------- | ------------------ | --------------------- | ------------------------------------ |
+| 1.7.x       | :white_check_mark: | Long Term Support     | See [Long Term Support policy][0]    |
```
Comment (mgol) [inline SECURITY.md:7]: 1.7 will no longer be supported when 1.8.0 is getting released which should happen soon.

Comment (Splaktar) [reply SECURITY.md:7]: Agreed, but we should probably update this at that time?

Comment (mgol) [reply SECURITY.md:7]: I’ll leave it to @petebacondarwin, I just wanted to make sure we won’t miss this.

Comment (mgol) [reply SECURITY.md:7]: It might be easy to miss if this PR & #17027 land independently.

**Comment by Splaktar** [comment]
Updated for 1.8.x

**Comment by Splaktar** [comment]
The failing test appears to be a flake. I am unable to re-run it.

**Comment by mgol** [comment]
@Splaktar I restarted the failing job.

**Comment by Splaktar** [comment]
All feedback has been addressed.

**Pull request merged**